### ISSUE TITLE
CSS Autocomplete Is Broken (v2.1.3)

### ISSUE THREAD

**Issue by VaelVictus** [braver/SublimeSass#30]
Prior to this update, I could type `dis` and I would be suggested `display`, and `bl` and it would suggest me only `block`. Now it's just a mess of anything remotely suggestible:

![image](https://user-images.githubusercontent.com/1727143/48302949-b160a980-e4d1-11e8-8630-2ae2cd35c24c.png)

I do recall there was a problem like this before and I'd had to mess with some config files, but I've just reinstalled both this and the [SCSS](https://github.com/MarioRicalde/SCSS.tmbundle) plugin and it's still not working.

**Comment by braver** [comment]
Things to try:
- double check those config files
- close all files, the restart ST
- re-assign the syntax to SCSS for each file
- wipe your sublime-workspace

**Comment by VaelVictus** [comment]
I've installed a completely new sublime text 3 and it still isn't working. I've always used portable ST's, so I've just now installed build 3176, up from using 3143. **This is a clean installation, not overwriting.** I imported my theme, Package Control, and then immediately:

Installed SCSS (by mario)
Installed this plugin

Does not work. I've triple-checked, I am definitely in the new version. I restarted sublime, opened up a new window.

Basically, my understanding is that all your plugin should do for autocompletes is look at the css autocompletes built into Sublime. I remember **in the other version, not my current** having to edit sublime's autocomplete definitions to simply allow .scss to be treated as .css and the completion worked as expected.

I'm attaching example behaviors.
![image](https://user-images.githubusercontent.com/1727143/48317514-72aa1c80-e5c1-11e8-8670-47465f1f9429.png)
